platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/internal/EventStreamPathIterator.java

```diff
@@ -96,18 +119,18 @@ private static boolean isFileAnEventStreamFile(final Path path) {
     }
 
     /**
-     * Look inside an event stream file and return the round of the first event.
+     * Look inside an event stream file and return the first event
      *
      * @param path
      * 		a path to an event stream file
      * @return the round of the first event in the file
      */
-    private static long getFirstRoundInEventStreamFile(final Path path) throws IOException {
+    private static DetailedConsensusEvent getFirstEventInEventStreamFile(final Path path) throws IOException {
```

Comment:
Return object and param can be annotated

Reply:
added